src/services/offerService.js

```diff
@@ -167,13 +180,16 @@ export const enableOffer = measureTime(TIMED_ACTIONS.OFFER_ENABLE, async (offerI
 
     } catch (error) {
 
-        if (!isErrorRegistered) createEvent(EVENT_TYPES.ERROR(
+        const errorArray = Array.isArray(error) ? error :
+            [{ msg: "Unexpected Error. Please try again later." }];
+
+        if (!isErrorRegistered) createErrorEvent(
```

Comment:
when the function call spans over multiple lines, it's better to have the `{}` around the if body